crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -1196,7 +1196,10 @@ pub async fn run(
             } else {
                 use crate::commands::run;
                 event.track_run_code_path(CodePath::Rust);
-                let exit_code = run::run(base, event).await?;
+                let exit_code = run::run(base, event).await.map_err(|e| {
+                    error!("run failed: {e}");
```

Comment:
~~Will this cause us to stutter our errors?~~ 

I see that this logic moved from the run command itself. I don't care where it lives, but I feel it's easier to reason about if both error messages live close to each other instead of being spread across two functions. Maybe use `inspect_err` in the same way we use `inspect`?

Reply:
Good suggestion. Moved to `inspect_err`.

Reply:
Eventually I want to remove all of this and have a single print out in `turborepo-lib/src/lib.rs` that prints the miette formatted version. That's in #6956